lib/scrabble.rb

```diff
@@ -15,4 +17,24 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    if word.nil?
+      @score
+    else
+      letters = word.upcase.split("")
+
+      letters.each do |letter|
+        @score += point_values[letter]
+      end
+    @score
+    end
+  end
+
+  def score_with_multipliers(word, letter_multiplier, word_multiplier =1)
+    letters = word.upcase.split("")
+    
```

Comment:
Watch out for extra spaces (I recommend giving your code a quick glance before you put in a final product to clean up things like this)